src/api/graphql.generated.tsx

```diff
@@ -1121,8 +1121,9 @@ export interface LoginInput {
 
```

Comment:
I think your schema file is outdated. This looks like it's reverting recent changes

Reply:
Hmm, I just did a git pull on the master branch in cord-api-v3 but don't see any changes in the schema file o_O

Reply:
DId you start the server?

Reply:
yeah I restarted the server after git pull

Reply:
Well I know `LoginOutput` doesn't have `success` anymore and you added it here...

Reply:
Okay I deleted the `schema.graphql` file, restarted api server, and that seems to have updated the generated file. But now I'm getting this error:
```
TypeScript error in /Users/richard/repo/open/cord-field/src/scenes/Authentication/ForgotPassword/ForgotPasswordForm.tsx(4,10):
Module '"../../../../../../../../../Users/richard/repo/open/cord-field/src/api"' has no exported member 'UserEmailInput'.  TS2305

    2 | import React from 'react';
    3 | import { Form, FormProps } from 'react-final-form';
  > 4 | import { UserEmailInput } from '../../../api';
```

Reply:
Can't seem to find any record of `UserEmailInput` in the server

Reply:
Ah I just changed that....

You could just use `{ email: string}` instead.

Reply:
👍